apps/starccm/run_case.pbs

```diff
@@ -2,26 +2,29 @@
 
 # parameters that can be overridden
 APP_INSTALL_DIR=${APP_INSTALL_DIR:-/apps}
-DATA_DIR={DATA_DIR:-.}
-CASE={CASE:-civil}
-# PODKEY is required (pass in as environment variable)
-#PODKEY=
+DATA_DIR=${DATA_DIR:-/data/starccm}
+CASE=${CASE:-civil}
+# Valid OMPI values are openmpi and openmpi4
```

Comment:
hmm.. isn't it just openmpi at this time? would suggest not mentioning this.

Reply:
Addressed comment